components/script/image_listener.rs

```diff
@@ -23,6 +23,7 @@ pub fn generate_cache_listener_for_element<
     T: ImageCacheListener + DerivedFrom<Node> + DomObject,
 >(
     elem: &T,
+    can_gc: CanGc
```

Comment:
Since the only CanGc use is within the task macro, we can remove this argument.

Reply:
I've pushed the updates